modules/serverless-logic-deploying-workflows-dev-mode.adoc

```diff
@@ -0,0 +1,137 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-logic/serverless-logic-creating-managing-workflows.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-logic-deploying-workflows-dev-mode"]
+= Deploying workflows in Dev mode
+
+You can deploy your local workflow on OpenShift in Dev mode. This allows you to experiment and modify your workflow directly on the cluster, seeing changes almost immediately. Dev mode is designed for development and testing purposes. It is ideal for initial development stages and testing new changes.
```

Comment:
Again, maybe use "OpenShift Container Platform", or ideally an attribute for it, instead of "OpenShift".

Also I would replace "This allows you to experiment..." with "You can use this deployment to experiment..."

And I would add a second "for" : "It is ideal for initial development stages and for testing new changes."